src/app/pages/transactions/in-error/transactions-in-error-table-data-source.ts

```diff
@@ -91,11 +95,57 @@ export class TransactionsInErrorTableDataSource extends TableDataSource<Transact
     });
   }
 
+  public toggleRowSelection(row: Data, event: MatCheckboxChange) {
```

Comment:
Call super class and add just the line to disable the button

Reply:
Done